README.md

```diff
@@ -1,2 +1,21 @@
 # open-source-funders
-Name and descrition TBA
+
+> Name and description TBA
+
+## Background
+
+Funding in open source is a continual problem. For a project to survive beyond the momentary interest of its maintainer, it needs funds. Today, there are more ways of funding open source projects and contributors than ever: GitHub Sponsors, Open Collective, `npm fund`, Patreon, and so on. Thousands of projects have made it clear they need funds; hundreds of sponsors have helped out. But it's not always clear which companies sponsor Open Source. It's also not obvious which companies sponsor who, and on what platform.
+
+This project highlights companies that have funded the open source ecosystem. We showcase companies that have given money to open source projects through any open source platform, and we monitor the different ways they do so: either through direct payouts, sponsorships, hiring engineers to work full or part time on open source, funding meetups or conferences, and so on.
```

Comment:
`through any open source platform`, does that exclude patreon etc? `Through several platforms` maybe?